### DISCUSSION TITLE
Running and pending to run cells

### DISCUSSION THREAD

**Discussion by aytugkaya** [microsoft/vscode-jupyter#9726] [Questions and Answers] [answered]
Jupyter server marks all cells that are being computed or have been marked for next cell computation with an asterisk on the left side of the cell as [*].

For example, if I hit Run All button for a notebook, all cells should be marked [*] and as a cell is computed it should be converted to [number].

However, with VSCode Jupyter extension, the [*] can't be observed. This is causing confusion, especially if more than one cells are selected for execution, where cells pending execution will be marked [ ]. It is the same notation for unexecuted cells.

**Comment by amunger** [comment, marked as answer]
There should be a visual queue for pending cells, underlined in the screenshot below. Are you seeing that when running cells? 

![image](https://user-images.githubusercontent.com/2019016/164538463-1cae5bd5-4088-4ed7-b9e3-159e36260fa7.png)
